packages/lab/src/combo-box-next/ComboBoxNext.tsx

```diff
@@ -0,0 +1,241 @@
+import {
+  ChangeEvent, ComponentPropsWithoutRef,
+  FocusEvent,
+  forwardRef,
+  InputHTMLAttributes,
+  ReactNode,
+  Ref,
+  SyntheticEvent,
+  useRef,
+} from "react";
+import {
+  Input,
+  makePrefixer,
+  SaltProvider,
+  useForkRef,
+  useId,
+} from "@salt-ds/core";
+import { ListNext, ListNextProps } from "../list-next";
+import { FloatingPortal } from "@floating-ui/react";
+import { useComboBox } from "./useComboBox";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import comboBoxNextCss from "./ComboBoxNext.css";
+import { ChevronDownIcon, ChevronUpIcon } from "@salt-ds/icons";
+import { defaultFilter, defaultItemRenderer } from "./utils";
+import { clsx } from "clsx";
+import { UsePortalProps } from "./useComboboxPortal";
+
+const withBaseName = makePrefixer("saltComboBoxNext");
+
+export interface ComboBoxNextProps extends Omit<ComponentPropsWithoutRef<"input">, "onChange"> {
+  /**
+   * [Attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Attributes) applied to the `input` element.
+   */
+  InputProps?: InputHTMLAttributes<HTMLInputElement>;
```

Comment:
should these actually be InputProps type of our component, like for List below? seems inputProps is also given to the Input, otherwise here would be inputProps